elide-core/src/main/java/com/yahoo/elide/core/DataStoreTransaction.java

```diff
@@ -139,4 +139,28 @@ default void flush() {
                                                           Optional<Sorting> sorting, Optional<Pagination> pagination) {
         return collection;
     }
+
+    default <T> Object getRelation(
+            Object entity,
+            RelationshipType relationshipType,
+            String relationName,
+            Class<T> relationClass,
+            EntityDictionary dictionary,
+            Set<Predicate> filters
+    ) {
+        return PersistentResource.getValue(entity, relationName, dictionary);
```

Comment:
Acutally, I have a question. Why are we passing `Object entity` rather than `PersistentResource resource` and then just calling `resource.getRelation` directly. We're trying to express our current assumption of everything being a proxy, right, so let's just call the method on the proxy.